core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.ts

```diff
@@ -1,95 +1,85 @@
+import { EditContentPayload } from '../../models/dot-edit-content-form.interface';
 import { DotEditContentService } from '../../services/dot-edit-content.service';
 
 @Component({
     selector: 'dot-edit-content-form-layout',
     standalone: true,
     imports: [
         NgIf,
+        JsonPipe,
         AsyncPipe,
         DotMessagePipe,
-        DotEditContentFormComponent,
         ButtonModule,
+        ToastModule,
+        DotEditContentFormComponent,
         DotEditContentAsideComponent,
-        JsonPipe
+        DotEditContentToolbarComponent
     ],
     templateUrl: './edit-content.layout.component.html',
     styleUrls: ['./edit-content.layout.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush,
-    providers: [DotEditContentService]
+    providers: [
+        DotWorkflowsActionsService,
+        DotWorkflowActionsFireService,
+        DotEditContentService,
+        DotMessageService,
+        MessageService,
+        DotEditContentStore
+    ]
 })
 export class EditContentLayoutComponent implements OnInit {
-    private activatedRoute = inject(ActivatedRoute);
-
-    public contentType = this.activatedRoute.snapshot.params['contentType'];
-    public identifier = this.activatedRoute.snapshot.params['id'];
+    private readonly activatedRoute = inject(ActivatedRoute);
+    private readonly store = inject(DotEditContentStore);
 
-    private readonly dotEditContentService = inject(DotEditContentService);
-    private formValue: Record<string, string>;
-    isContentSaved = false;
+    contentType = this.activatedRoute.snapshot.params['contentType'];
+    inode = this.activatedRoute.snapshot.params['id'];
+    formValue: Record<string, string>;
 
-    formData$: Observable<EditContentFormData> = this.identifier
-        ? this.dotEditContentService.getContentById(this.identifier).pipe(
-              switchMap(({ contentType, ...contentData }) => {
-                  if (contentType) {
-                      this.contentType = contentType;
-                      this.dotEditContentService.currentContentType = contentType;
-
-                      return this.dotEditContentService.getContentTypeFormData(contentType).pipe(
-                          map(({ layout, fields }) => ({
-                              contentlet: { ...contentData },
-                              layout,
-                              fields,
-                              contentType
-                          }))
-                      );
-                  } else {
-                      return EMPTY;
-                  }
-              })
-          )
-        : this.dotEditContentService
-              .getContentTypeFormData(this.contentType)
-              .pipe(
-                  map(({ layout, fields }) => ({ layout, fields, contentType: this.contentType }))
-              );
+    vm$: Observable<EditContentPayload> = this.store.vm$.pipe(
+        skip(1),
```

Comment:
Maybe you can avoid this skip by lazy loading the store.

Reply:
Hujummm..

Reply:
Done!